alchemy/src/cloudflare/wrangler.json.ts

```diff
@@ -620,4 +633,8 @@ function processBindings(
   if (pipelines.length > 0) {
     spec.pipelines = pipelines;
   }
+
+  // if (secretsStoreSecrets.length > 0) {
+  //   spec.secrets_store_secrets = secretsStoreSecrets;
+  // }
```

Comment:
we need this, but you also n eed to make sure to build up the array when iterating through the bindings